src/map-view.android.ts

```diff
@@ -133,6 +134,45 @@ export class MapView extends MapViewBase {
                     onMyLocationButtonClick: () => {
                         owner.notifyMyLocationTapped();
 
+                        return false;
+                    },
+                }));
+
+                gMap.setOnIndoorStateChangeListener(new com.google.android.gms.maps.GoogleMap.OnIndoorStateChangeListener({
+                    onIndoorBuildingFocused: () => {
+                        const buildingFocused = gMap.getFocusedBuilding();
+                        const levels = [];
```

Comment:
buildingFocused will be null sometimes (when moving away from a floorplan)

Reply:
Oh thanks! I've added a condition on it to return null if we got nothing